docs/docs/how-tos/nebari-aws.md

```diff
@@ -152,6 +152,17 @@ With the `qhub-config.yaml` configuration file now created, Nebari can be deploy
 ```bash
 qhub deploy -c qhub-config.yaml
 ```
+:::note
+If you have account on any [Cloud provider](https://www.nebari.dev/getting-started/cloud-providers) you can create a domain name of your choice, if you are not selecting CloudFlare for [automatic DNS registration](https://www.nebari.dev/how-tos/domain-registry). Taking Cloud Flare as a choice you can create a domain name with some restrictions obviously, for example: `any-domain-name.qhub.dev`. Please keep in mind that after creating your own domain name you need to run  
```

Comment:
>If you have account on any [Cloud provider](https://www.nebari.dev/getting-started/cloud-providers) you can create a domain name of your choice, if you are not selecting CloudFlare for [automatic DNS registration](https://www.nebari.dev/how-tos/domain-registry).

I don't understand this sentence. If I'm setting up Nebari on AWS, then I will have an AWS account. Also, I can only create a domain name of my choice if I have cloud provider account? But Cloud Flare is more restrictive? 

```suggestion
The default option in Nebari uses CloudFare for [automatic DNS registration](https://www.nebari.dev/how-tos/domain-registry). If you choose CloudFlare, you can create a domain name with some restrictions, for example: `any-domain-name.qhub.dev` (can we just delete this sentence - or, what are the restrictions and why does cloudflare have them?). After creating your own domain name, you will need to run  
```

Reply:
> > If you have account on any [Cloud provider](https://www.nebari.dev/getting-started/cloud-providers) you can create a domain name of your choice, if you are not selecting CloudFlare for [automatic DNS registration](https://www.nebari.dev/how-tos/domain-registry).
> 
> I don't understand this sentence. If I'm setting up Nebari on AWS, then I will have an AWS account. Also, I can only create a domain name of my choice if I have cloud provider account? But Cloud Flare is more restrictive?

If you don't have an account on cloudflare and don't have access to create a domain name then you need to follow some restrictions like `qhub.dev` in the end while executing `qhub init` command in the domain name part. else if you have an account and can create a domain name then you can create any domain name of your choice. This is the logic behind it. @viniciusdc please correct me, if I'm wrong.

Reply:
So CloudFlare is the default, but you don't have to have an account to use it?

If you use CloudFlare without an account, you have to have use the suffix `qhub.dev`? Why?

If you use CloudFlare with an account, you can choose any domain name you want. Right?

Reply:
> So CloudFlare is the default, but you don't have to have an account to use it?
> 
> If you use CloudFlare without an account, you have to have use the suffix `qhub.dev`? Why?
> 
> If you use CloudFlare with an account, you can choose any domain name you want. Right?

Yeah. That's confusing. I'm changing this note in perspective of all DNS provider and Adding useful information regarding in `nebari DNS section`.